src/styles/card/blocks/card.scss

```diff
@@ -0,0 +1,85 @@
+.card {
+  border: 1px solid $color-gray;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition: all 0.3s;
+
+  &__description {
+    color: $text-main-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &:hover {
+    transform: scale(120%);
```

Comment:
The `transform: scale(120%);` property should use decimal notation for scaling, such as `transform: scale(1.2);`, to ensure consistency and avoid potential issues with different browsers.